financial-services-accelerator/components/org.wso2.financial.services.accelerator.event.notifications.service/src/main/java/org/wso2/financial/services/accelerator/event/notifications/service/EventPollingService.java

```diff
@@ -0,0 +1,161 @@
+                                eventPolling.getClientId(), EventNotificationConstants.OPEN, setsToReturn);
+                    }
+
+                    if (notificationList.isEmpty()) {
+                        if (log.isDebugEnabled()) {
+                            log.debug(String.format("No FS Event Notifications available for for the client " +
+                                    "with ID : '%s'.", eventPolling.getClientId().replaceAll("[\r\n]", "")));
+                        }
+                        aggregatedPollingResponse.setStatus(EventNotificationConstants.NOT_FOUND);
+                    } else {
+                        if (log.isDebugEnabled()) {
+                            log.debug(String.format("FS Event Notifications available for the client " +
+                                    "with ID : '%s'.", eventPolling.getClientId().replaceAll("[\r\n]", "")));
+                        }
+                        aggregatedPollingResponse.setStatus(EventNotificationConstants.OK);
+
+                        for (Notification notification : notificationList) {
+
+                            //Get events by notificationId
+                            List<NotificationEvent> notificationEvents = eventNotificationDAO.
+                                    getEventsByNotificationID(connection, notification.getNotificationId());
+
+                            NotificationResponse responseNotification = eventNotificationGenerator.
+                                    generateEventNotificationBody(notification, notificationEvents);
+                            sets.put(notification.getNotificationId(), eventNotificationGenerator
+                                    .generateEventNotification(NotificationResponse.getJsonNode(responseNotification)));
+                            log.info("Retrieved FS event notifications");
+                        }
+                        aggregatedPollingResponse.setSets(sets);
+                    }
+                }
+
+                int count = eventNotificationDAO.getNotificationCountByClientIdAndStatus(connection,
+                        eventPolling.getClientId(), EventNotificationConstants.OPEN)
+                        - aggregatedPollingResponse.getSets().size();
+
+                aggregatedPollingResponse.setCount(count);
+                DatabaseUtils.commitTransaction(connection);
+
+                return aggregatedPollingResponse;
+            } catch (FSEventNotificationException |
+                     IOException | JOSEException | IdentityOAuth2Exception e) {
+                log.debug("Error when retrieving FS event notifications.", e);
+                DatabaseUtils.rollbackTransaction(connection);
+                throw new FSEventNotificationException("Error when retrieving FS event notifications.", e);
+            } finally {
+                log.debug(EventNotificationConstants.DATABASE_CONNECTION_CLOSE_LOG_MSG);
```

Comment:
Some classes are missing this debug log. Shall we make it consistent?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/159/commits/4e3160f15e06259be81743872bbf3ecd1c33db7e